src/runtime/builtin_fp16.cc

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) 2018 by Contributors
+ * \file builtin_fp16.cc
+ * \brief Functions for conversion between fp32 and fp16
+*/
+
+#include <builtin_fp16.h>
+
+namespace tvm {
+namespace runtime {
+
+extern "C"  uint16_t __gnu_f2h_ieee(float a) {
```

Comment:
Followup comment, we need to set it to be TVM_WEAK linkage so if user link compiler-rt, there will not be type conflict